projects/core/src/lib/backend/backend.model.ts

```diff
@@ -41,6 +41,16 @@ export interface BBoxDict {
     upperRightCoordinate: CoordinateDict;
 }
 
+export interface SpatialPartitionDict {
+    upperLeftCoordinate: CoordinateDict;
+    lowerRightCoordinate: CoordinateDict;
+}
+
+export interface SpatialResolution {
+    x: number;
+    y: number;
+}
+
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/206677590-48bcf7ff-a373-4144-8505-198e2739344b.png)

There is somehow an error in the console, when I use this component.

Reply:
[655b179](https://github.com/geo-engine/geoengine-ui/pull/532/commits/655b17919e1aed11a2ce17ff7289dfc445b3e2d5)